Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/rest/create_test.go

```diff
@@ -25,17 +25,17 @@ import (
 
 func TestCheckGeneratedNameError(t *testing.T) {
 	expect := errors.NewNotFound("foo", "bar")
-	if err := CheckGeneratedNameError(Pods, expect, &api.Pod{}); err != expect {
+	if err := CheckGeneratedNameError(Services, expect, &api.Pod{}); err != expect {
```

Comment:
Is it correct to be using the services strategy when validating pod name generation?  In controllers/types.go they're  technically the same strategy, just struck me as strange when reading it.